modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+In a cluster, as long as a database is in a shutdown state, it can not be considered available to other members of the cluster.
+It is not possible to do online backups against shutdown databases and they need to be taken into special consideration during disaster recovery, as they do not have a running Raft machine while shutdown.
+Dropped databases are completely removed and are not intended to be used again at all.
+====
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-start]]
+== Start databases
+
+Databases can be started using the command `START DATABASE`.
+
+.Query
+[source, cypher]
+----
+START DATABASE customers
+----
+
+[NOTE]
+====
+Both standard databases and composite databases can be stopped using this command.
+====
+
+The status of the started database can be seen using the command `SHOW DATABASE name`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE customers YIELD name, requestedStatus, currentStatus
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------+
+| name        | requestedStatus | currentStatus |
++-----------------------------------------------+
+| "customers" | "online"        | "online"      |
++-----------------------------------------------+
+----
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-delete]]
+== Delete databases
+
+Databases can be deleted by using the command `DROP DATABASE`.
+
+.Query
+[source, cypher]
+----
+DROP DATABASE customers
+----
+
+When a database has been deleted, it no longer shows up in the listing provided by the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name
+----
+
+.Result
+[role="queryresult]
+----
++----------+
+| name     |
++----------+
+| "movies" |
+| "neo4j"  |
+| "system" |
++----------+
+----
+
+The `DROP DATABASE` command is optionally idempotent, with the default behavior to fail with an error if the database does not exist.
+Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the database not exist.
+It will always return an error if there is an existing alias that targets the database.
+In that case, the alias needs to be dropped before dropping the database.
+
+.Query
+[source, cypher]
+----
+DROP DATABASE customers IF EXISTS
+----
+
+The `DROP DATABASE` command will remove a database entirely.
+
+[role=enterprise-edition not-on-aura]
+== Dump and delete databases
+
+You can request that a dump of the store files is produced first, and stored in the path configured using the `dbms.directories.dumps.root` setting (by default `<neo4j-home>/data/dumps`).
+This can be achieved by appending `DUMP DATA` to the command (or `DESTROY DATA` to explicitly request the default behavior).
+These dumps are equivalent to those produced by `neo4j-admin dump` and can be similarly restored using `neo4j-admin load`.
+
+Note that all database aliases must be dropped before dropping a database.
```

Comment:
I feel like this sentence should be further up in the section, maybe close to the `The `DROP DATABASE` command will remove a database entirely.` line?

Reply:
though I guess it goes with the test below, though this particular thing has nothing to do with the dump/destroy data part of it, all aliases needs to be dropped beforehand regardless 🤷

Reply:
Yeah I still think saying that all aliases needs to be dropped first should not be hidden in the dump/destroy data section but be part of the initial section